core/common/tsconfig.json

```
@@ -6,7 +6,7 @@
     "outDir": "./lib"
   },
   "include": [
-    "./src/**/*.ts"
+    "./src/**/*.ts",
```

JSONs generally don't support trailing commas and often tools fail to parse them. Not sure if `tsc` allows it, but I guess for consistency it should be removed.
```suggestion
    "./src/**/*.ts"
```